doc/hop.txt

```diff
@@ -589,6 +595,28 @@ below.
     Defaults:~
         `direction = nil`
 
+`multi_windows`                                                   *multi_windows*
+    Enable hop in multiple windows or not.
+
+    Defaults:~
+        `multi_windows = false`
+
+`dict_list`
```

Comment:
The name of this configuration variable is not explicit enough. Can you change it to something that conveys what it’s used for?

Reply:
How about `match_mappings` or `search_mappings`? Means that extending the match space via mapping the pattern to what `match_mappings` provided.